src/helm/proxy/clients/together_client.py

```diff
@@ -241,7 +240,28 @@ def do_it_async() -> Dict[Any, Any]:
                 job_id = submit_job()
                 return retrieve_job(job_id)
 
-            response, cached = self.cache.get(cache_key, wrap_request_time(do_it_async))
+            def do_it_sync() -> Dict[Any, Any]:
```

Comment:
Together's inference v2 API doesn't support async, so using sync.

Reply:
Can just revert this file; #2093 disables async.